neuland/calibration/R3BNeulandTSyncer.cxx

```diff
@@ -0,0 +1,345 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019 Members of R3B Collaboration                          *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+#include "R3BNeulandTSyncer.h"
+#include "LSQR.h"
```

Comment:
A small comment:
At first i wanted to remove it.
After doing so and using the root way, i noticed this takes way longer.
It will be even worse with more planes...
So I at least rewrote the wrapper around it.